dev/com.ibm.ws.classloading/src/com/ibm/ws/classloading/internal/AppClassLoader.java

```diff
@@ -71,11 +74,63 @@
  * to discover the special methods:
  */
 public class AppClassLoader extends ContainerClassLoader implements SpringLoader {
+    static final TraceComponent tc = Tr.register(AppClassLoader.class);
+
+    private static final Set<String> forbiddenClasses = new HashSet<>();
```

Comment:
Make this an unmodifiable set in the static block below instead of assigning it to a modifiable set here.